packages/components/src/local/atoms/text-input/types/props.d.ts

```diff
@@ -7,14 +7,26 @@ export default interface PropTypes {
    * The label of the text input
    */
   label?: string
+  /**
+   * The color of the input label
+   */
+  labelColor?: string
+  /**
+   * The size of the input label
+   */
+  labelSize?: string | number
+  /**
+   * Text field variants
+   */
+  variant?: 'standard' | 'filled' | 'outlined'
   /**
    * The value of the input
    */
-  value: number | string
+  value?: number | string
   /**
    * The number of lines the input should have (default 1)
```

Comment:
I think the comment needs updating, to explain the boolean meaning.

Reply:
value props explained.